src/components/shared/StudiesTable.tsx

```diff
@@ -50,14 +50,14 @@ export default function StudiesTable(props: StudiesTableProps) {
         }
     }, [activePage, column, direction, isMobileDevice, pageLength, props.dataRecords, searchResults])
 
-    const getPopulation = (sex: string | null, age: string[] | null, population_group: string[] | null) => {
+    const getPopulation = (sex: string | null, age: string | null, population_group: string | null) => {
         // TODO: Check if we still need to accommodate for these edge cases.
         if (!population_group) {
             return "Not Reported";
         }
         const displaySex = sex && sex !== "All" && sex !== "Unspecified";
-        const displayAge = age && age[0] !== "All" && age[0] !== "Unspecified";
```

Comment:
Is age being "All" or "Unspecified" not possible anymore?

Reply:
I think Unspecified might still be possible but All has been changed to Multiple groups. I'll readd the Unspecified check